guide/14-deep-learning/How_MaskRCNN_works.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Octothorpes (##) should be removed.</li></ul><p> </p><ul><li><em>over-smoothen&nbsp;</em>does not sound right in the first sentence. I think it sounds better to say:</li></ul><p>						<em>Segmentation models can tend to over generalize boundaries </em>								<em>which might not be precise for objects or scenes with irregular boundaries.</em></p><p>Or:</p><p>						<em>Segmentation models can tend to remove to many details from boundaries which might not be precise for objects or scenes with irregular boundaries.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I've moved the "PointRend Enhancement" to a different cell. Keeping the (###) to format it as a sub-heading.</p>

Reply:
<p>"Smooth boundaries" is a common term in this context and has a meaning that "generalize boundaries" doesn't convey. "Over" is used to convey that it is more than what it needs to be. The abstract (https://arxiv.org/abs/1912.08193) uses "...boundaries in regions that are <em>over-smoothed</em> by previous methods."</p><p> </p><p>Changed the sentence to <em>Segmentation models can tend to generate over-smooth boundaries which might not be precise for objects or scenes with irregular boundaries.</em></p>